fe/src/main/java/org/apache/doris/analysis/ExpressionFunctions.java

```diff
@@ -144,15 +145,6 @@ private synchronized void registerFunctions() {
             }
         }
         this.functions = mapBuilder.build();
-
-        // Functions that need to handle null.
-        ImmutableSet.Builder<String> setBuilder =
-                new ImmutableSet.Builder<String>();
-        setBuilder.add("if");
-        setBuilder.add("hll_hash");
-        setBuilder.add("concat_ws");
-        setBuilder.add("ifnull");
-        this.nonNullResultWithNullParamFunctions = setBuilder.build();
```

Comment:
Why move this to `gensrc/script/doris_builtins_functions.py`?  what's the advantage?

Reply:
The previous definition was hidden in `ExpressionFunctions`. If it weren't for this bug, I wouldn't even know that there is such a definition. So I decided to put this definition in `doris_builtins_functions.py`, so that if there are new builtin functions added later, they can be unified in `doris_builtins_functions.py` to define its behavior.

Reply:
I See.

But if this change is only to make the `nonNullResultWithNullParamFunctions` definition Obviously. I think normal java class is better than python script, because current IDE is very powerful, we could very easily find the definition.

Besides，I think if not performance reason, we shouldn't  use code gen.